origin-dapp/src/services/origin.js

```diff
@@ -10,12 +10,9 @@ import OrbitDB from 'orbit-db'
  * But Micah couldn't get it to connect ¯\_(ツ)_/¯
  */
 const defaultProviderUrl = process.env.PROVIDER_URL
-const defaultBridgeUrl = 'https://bridge.originprotocol.com'
 const bridgeProtocol = process.env.BRIDGE_SERVER_PROTOCOL
 const bridgeDomain = process.env.BRIDGE_SERVER_DOMAIN
-const customBridgeUrl = `${bridgeProtocol}://${bridgeDomain}`
-const hasCustomBridge = bridgeProtocol && bridgeDomain
-const bridgeUrl = hasCustomBridge ? customBridgeUrl : defaultBridgeUrl
+const bridgeUrl = `${bridgeProtocol}://${bridgeDomain}`
```

Comment:
As opposed to using 2 env vars, one for PROTOCOL and the other for DOMAIN, I'd be in favor of a single var call URL that stores both. This is not only for here, I'm thinking as a general thing for our codebase. We don't need to fix it in this PR though.

Reply:
Yea agreed, I think this is this way because it is easier to display it in the connectivity dropdown without having to parse a URL and extract the hostname. A similar thing applies to the IPFS settings where there are four environment variables to set that should be replaced with just a URL.